### ISSUE TITLE
Unable to install: No such file or directory

### ISSUE THREAD

**Issue by theokeist** [NicholasBHubbard/sbozyp#5]
```

root@darkstar:~/Downloads# VERSION=0.0.6

root@darkstar:~/Downloads# wget https://cpan.metacpan.org/authors/id/N/NH/NHUBBARD/App-sbozyp-$VERSION.tar.gz
--2024-10-28 19:51:26--  https://cpan.metacpan.org/authors/id/N/NH/NHUBBARD/App-sbozyp-0.0.6.tar.gz
Resolving cpan.metacpan.org (cpan.metacpan.org)... 146.75.118.217, 2a04:4e42:39::729
Connecting to cpan.metacpan.org (cpan.metacpan.org)|146.75.118.217|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 24195 (24K) [application/x-gzip]
Saving to: ‘App-sbozyp-0.0.6.tar.gz’

App-sbozyp-0.0.6.t 100%[================>]  23.63K  --.-KB/s    in 0.005s  

2024-10-28 19:51:26 (4.20 MB/s) - ‘App-sbozyp-0.0.6.tar.gz’ saved [24195/24195]

root@darkstar:~/Downloads# tar -xf App-sbozyp-0.0.6.tar.gz 

root@darkstar:~/Downloads# chmod +x App-sbozyp-0.0.6/package/sbozyp.SlackBuild 

root@darkstar:~/Downloads# App-sbozyp-0.0.6/package/sbozyp.SlackBuild 
Checking if your kit is complete...
Looks good
Generating a Unix-style Makefile
Writing Makefile for App::sbozyp
Writing MYMETA.yml and MYMETA.json
cp lib/App/sbozyp.pm blib/lib/App/sbozyp.pm
cp bin/sbozyp blib/script/sbozyp
"/usr/bin/perl5.40.0" -MExtUtils::MY -e 'MY->fixin(shift)' -- blib/script/sbozyp
Manifying 1 pod document
Manifying 1 pod document
Manifying 1 pod document
Manifying 1 pod document
Installing /tmp/build/package-sbozyp/usr/share/perl5/vendor_perl/App/sbozyp.pm
Installing /tmp/build/package-sbozyp/usr/man/man1/sbozyp.1
Installing /tmp/build/package-sbozyp/usr/man/man3/App::sbozyp.3
Installing /tmp/build/package-sbozyp/usr/bin/sbozyp
Appending installation info to /tmp/build/package-sbozyp/usr/lib64/perl5/perllocal.pod

cp: cannot stat 'sbozyp.SlackBuild': No such file or directory

```

**Comment by theokeist** [comment]
It is fresh current installation

**Comment by NicholasBHubbard** [comment]
Was able to reproduce, thanks. Released a new version for this problem (`0.0.7`). Let me know if this fixes it.
